### DISCUSSION TITLE
hrp 在性能测试方面跟 JMeter、LoadRunner、Locust 有什么区别？

### DISCUSSION THREAD

**Discussion by zhuifengshen** [httprunner/httprunner#1184] [2 - 答疑（Q&A）]
性能测试方面 hrp 除了基于 API 脚本，其他方面跟市面主流的 JMeter、LoadRunner、Locust 有什么区别呢？

**Comment by xulz** [comment]
楼主可以查看下该介绍页 [整体概览](https://httprunner.com/docs/overview/)。
个人理解： 
hrp的性能会更好，locust被python拖累，对高性能有需求的不太适用。

hrp还是新项目，你可以保持关注

**Reply by debugtalk** [reply]
感谢解答。这个问题我会单独写一篇文章进行详细对比说明。
